Revit_oM/Parameters/RevitParameter.cs

```diff
@@ -43,9 +43,15 @@ public class RevitParameter : IObject
         [Description("Unit type of the Revit parameter.")]
         public virtual string UnitType { get; set; }
 
+        [Description("Current Using Unit of the Revit parameter.")]
+        public virtual string DisplayUnit { get; set; }
+
         [Description("Whether the parameter is read only or modifiable by the Revit user.")]
         public virtual bool IsReadOnly { get; set; } = false;
 
+        [Description("Whether the parameter is instance parameter.")]
```

Comment:
This should not be needed, type parameters are already fetched and stored on the property correspondent to element type, e.g. for class `Wall` it is `Construction`:
![image](https://github.com/user-attachments/assets/89f0ead1-ea45-455c-940f-dc73d19a90bd)